### DISCUSSION TITLE
OTA Prebuild Firmware for SML Devices

### DISCUSSION THREAD

**Discussion by svalcken** [arendst/Tasmota#20903] [Support request] [answered]
Please Add prebuild OTA Images for the SML Devices. Thanks.

Here additional an idea for language support. Firmware only in English and then a additional file with translations include tag for supported version. With this the count of prebuild firmwares can be reduct.

**Comment by sfromis** [comment, marked as answer]
The Smart Meter Interface feature should be included in the "unofficial" dev scripting build available here:
https://github.com/tasmota/install/tree/firmware/firmware/unofficial

It would be a major change to externalize the translation string into a separate file, and not feasible to load strings into RAM instead of being part of the flashed image. Would also lead to increased code size, which is not great when Tasmota would be workable on an ESP8266 device with 1 MB of flash.
